pybullet build time: Jan  4 2023 16:46:11
Traceback (most recent call last):
  File "/Users/chaseduvall/Desktop/396 ALife/mybots/simulate.py", line 8, in <module>
    simulation.Run()
  File "/Users/chaseduvall/Desktop/396 ALife/mybots/simulation.py", line 37, in Run
    self.robot.Sense(i)
  File "/Users/chaseduvall/Desktop/396 ALife/mybots/robot.py", line 38, in Sense
    sensor.Get_Value(t)
  File "/Users/chaseduvall/Desktop/396 ALife/mybots/sensor.py", line 15, in Get_Value
    self.values[t] = pyrosim.Get_Touch_Sensor_Value_For_Link(self.linkName)
  File "/Users/chaseduvall/Desktop/396 ALife/mybots/pyrosim/pyrosim.py", line 53, in Get_Touch_Sensor_Value_For_Link
    for pt in pts:
TypeError: 'NoneType' object is not iterable
Exception ignored in: <function SIMULATION.__del__ at 0x7fbf488ceca0>
Traceback (most recent call last):
  File "/Users/chaseduvall/Desktop/396 ALife/mybots/simulation.py", line 54, in __del__
pybullet.error: Not connected to physics server.
